package main

import "testing"

func TestVC(t *testing.T) {

	// If there is an error in main this test will fail
	main()
}
